packages/logger/README.md

```diff
@@ -148,19 +166,19 @@ yarn;
 To run the `usage` example, set up FluentD first:
 
 ```bash
-npm run svc:fluent;
+yarn svc:fluent;
 ```
 
 In another terminal, run the usage application which will pipe to the FluentD instance:
 
 ```bash
-npm run demo-usage;
+yarn demo-usage;
```

Comment:
this doesn't work any longer

```
examples/usage/index.ts(4,35): error TS2345: Argument of type '{ namespace: string; }' is not assignable to parameter of type 'ILoggerOptions | undefined'.
  Object literal may only specify known properties, and 'namespace' does not exist in type 'ILoggerOptions | undefined'.
examples/usage/index.ts(15,3): error TS2345: Argument of type '{ namespace: string; transports: TransportStream[]; }' is not assignable to parameter of type 'ILoggerOptions | undefined'.
  Object literal may only specify known properties, and 'namespace' does not exist in type 'ILoggerOptions | undefined'.
examples/usage/index.ts(38,3): error TS2345: Argument of type '{ formatters: ((info: TransformableInfo) => { message: string; level: string; })[]; namespace: st...' is not assignable to parameter of type 'ILoggerOptions | undefined'.
  Object literal may only specify known properties, and 'namespace' does not exist in type 'ILoggerOptions | undefined'.
```

Reply:
ok wait, my bad, the build failed

Reply:
(it's all okay now, closing)

Reply:
xD

Reply:
i just realised i could've deleted it lol